projects/stargazer/plugins/other/radius/radius.cpp

```diff
@@ -0,0 +1,19 @@
+#include "radius.h"
+
+using STG::RADIUS;
+
+extern "C" STG::Plugin* GetPlugin()
+{
+    static RADIUS plugin;
+    return &plugin;
+}
+
+std::string RADIUS::GetVersion() const
+{
+    return "Radius authorizator v.1.0";
```

Comment:
Please remove 'authorizator'. This is not an authorization plugin, although it can optionally perform authorization. Just 'Radius' is fine.